contracts/src/v0.8/dev/ArbitrumValidator.sol

```diff
@@ -10,7 +10,7 @@ import "../SimpleWriteAccessController.sol";
 /* ./dev dependencies - to be moved from ./dev after audit */
 import "./interfaces/ArbitrumSequencerUptimeFeedInterface.sol";
 import "./interfaces/FlagsInterface.sol";
-import "./vendor/arb-bridge-eth/v0.8.0-custom/contracts/bridge/interfaces/IInbox.sol";
+import "./vendor/arbitrum/IArbitrumDelayedInbox.sol";
 import "./vendor/arb-bridge-eth/v0.8.0-custom/contracts/libraries/AddressAliasHelper.sol";
 import "./vendor/arb-os/e8d9696f21/contracts/arbos/builtin/ArbSys.sol";
```

Comment:
Please check the difference between these three imports.

The vendor structure was added because it's helpful to discover the origin of the vendor source files, even (especially) when they are duplicated in our codebase.

Reply:
The `ArbitrumValidator` contract currently points to a proxy contract that  implements the `calculateRetryableSubmissionFee` function.  This function is unfortunately not included in `IInbox` so we have defined our own interface named `IArbitrumDelayedInbox` here so that we don't have to import the entire contract.  Is there a better place to put `IArbitrumDelayedInbox`?

Here is the function we are trying to call: https://github.com/OffchainLabs/nitro/blob/ccbed4ae0ae31f5e73e0f6c0871f074037a3a12a/contracts/src/bridge/Inbox.sol#L262-L269

Reply:
moved `IArbitrumDelayedInbox` to interfaces folder.